### ISSUE TITLE
CalendarDay is missing aria-current attribute for today's date

### ISSUE THREAD

**Issue by CrossTheDev** [airbnb/react-dates#2009]
An accessibility attribute `aria-current="date"` is missing on the `CalendarDay` of today's date. It marks the current date to screen reader users.

https://www.w3.org/TR/wai-aria-1.1/#aria-current
